5.2.slice_of_cake.py

```diff
@@ -0,0 +1,15 @@
+# The function receives a dictionary of ingredients - the name of the ingredient
+# and its price per 100 grams, an optional parameter of a list of products that we will ignore,
+# and for each component you will receive an argument bearing the name and the quantity in grams
+# that we want and return the price for buying all the ingredients
+def get_recipe_price(prices, optionals=[], **ingredients):
+    final_price = 0
+    for ingredient in ingredients:
+        final_price += prices[ingredient] * ingredients.get(ingredient) // 100
+    print(final_price)
```

Comment:
Seprate IO from logic - 
https://wiki.c2.com/?SeparateIoFromCalculation

Reply:
I fixed it.